lib/arrays.md

```diff
@@ -5,44 +5,65 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is an Array?
 
-Replace me with your response...
+An array is a list of items.
 
 # What are some examples of information that would be Arrays as opposed to some other data type?
 
-Replace me with your response...
+Team rosters would work well in an array, as would an ingredient list.
 
 # What is one way, using Ruby, to retrieve the 6th element in an Array? How about the 8th element? What happens if you try to retrieve the value of the _9999th_ element (or some element that doesn't exist in the array)?
 
-Replace me with your response...
+If your array is simply
+Days = ['Sunday','Monday','Tuesday','Wednesday','Thursday','Friday','Saturday']
+#you can simply
+Days[whatver day you want to call]
 
 # The previous question asks about finding, for example, the 6th element in an Array. Is it possible to find the **-6th** (Notice the negative symbol!) element in an Array? What does that even mean?
 
-Replace me with your response...
+Sure, with this method, Ruby would count down, starting at 0 then negative, backwards around the array until the right one.
 
-# How would you perform an operation on every element inside an Array?
+# ?How would you perform an operation on everey lement inside an Arrayy 
 
-Replace me with your response...
+I am really struggling with this.... am I trying to add bullets to a list of names?  or make a list of numbers to perform mathmatical opperations on?
```

Comment:
No problem. Let's say you have this array:

```ruby
names = ["Dan", "Mindy", "Sumeet"]
```

If I wanted you to just display _one_ of the names (Let's say the first one), then you could write `puts names[0]`. But what if I wanted you to display **each** name? What code would you write for that?

Note: You could definitely do this:

```ruby
puts names[0]
puts names[1]
puts names[2]
```

But that's not the answer I'm looking for :)

Once you figure that out, ask yourself if it would be any different if you had an array of numbers like `scores = [32, 45, 12]` and you wanted to display each score (like you had displayed each name). Is it different at all?

Then ask yourself how you would display each score along with `Score: ` in front of it, so it results in this:

```
Score: 32
Score: 45
Score: 12
```

Ask questions as they come to you–I'm here to help.

Summary of tasks:

- [ ] Code to display each name.
- [ ] Code to display each score. Is it any different?
- [ ] Code to display each score with the text "Score: " in front of it.